package/tests/test_api/test_apps.py

```diff
@@ -62,3 +62,18 @@ def test_reload_endpoint_return_304_when_content_has_not_changed(
         # when the etag has changed, the server will return a 200
         response = client.get("/api/reload", headers={"If-None-Match": "new etag"})
         assert response.status_code == 200
+
+
+class TestFaviconEndpoint:
+    @pytest.fixture
+    def test_client(self):
+        app = apps.create_api_app_from_project(mock.MagicMock(), autoreload=True)
+        return TestClient(app)
+
+    def test_favicon_endpoint(self, test_client):
+        response = test_client.get("/favicon.ico")
+        assert response.status_code == 200
+        assert response.headers["content-type"] in [
+            "image/x-icon",
+            "image/vnd.microsoft.icon",
```

Comment:
QQ - we have 2 content-types here, is it because of linux and windows tests?

Reply:
Here 'image/x-icon' is  commonly used MIME type for ICO files, some server configurations might use 'image/vnd.microsoft.icon' as ICO files content type, in fact our test setup uses 'image/vnd.microsoft.icon' as ICO files content type. So added both content type for checks.